bounties_api/std_bounties/utils.py

```diff
@@ -0,0 +1,83 @@
+from rest_framework import serializers
```

Comment:
@zoek1 would like to define this file as function utilities. Any ideas on a more distinct name for the file? Also, I think this should live in the bounties directory since that is the root. Thoughts?

Reply:
works for me!